pynest/examples/music_cont_out_proxy_example/README.md

```diff
@@ -0,0 +1,16 @@
+# Music_cont_out_proxy example
+
+## Requirements
++ MUSIC 1.1.15 or higher
++ NEST 2.10.0 or higher compiled with MPI and MUSIC
```

Comment:
Change to 2.12.1, as that's the earliest possible where this code will be made available.